datadog/resource_datadog_logs_index_order.go

```diff
@@ -51,24 +51,30 @@ func resourceDatadogLogsIndexOrderUpdate(d *schema.ResourceData, meta interface{
 	datadogClientV1 := providerConf.DatadogClientV1
 	authV1 := providerConf.AuthV1
 
-	if _, _, err := datadogClientV1.LogsIndexesApi.UpdateLogsIndexOrder(authV1).Body(ddIndexList).Execute(); err != nil {
+	updatedOrder, _, err := datadogClientV1.LogsIndexesApi.UpdateLogsIndexOrder(authV1).Body(ddIndexList).Execute()
+	if err != nil {
 		return translateClientError(err, "error updating logs index list")
 	}
 	d.SetId(tfId)
-	return resourceDatadogLogsIndexOrderRead(d, meta)
+	return updateLogsIndexOrderState(d, &updatedOrder)
+}
+
+func updateLogsIndexOrderState(d *schema.ResourceData, order *datadogV1.LogsIndexesOrder) error {
+	if err := d.Set("indexes", order.GetIndexNames()); err != nil {
+		return err
+	}
+	return nil
 }
 
 func resourceDatadogLogsIndexOrderRead(d *schema.ResourceData, meta interface{}) error {
 	providerConf := meta.(*ProviderConfiguration)
-	client := providerConf.CommunityClient
-	ddIndexList, err := client.GetLogsIndexList()
+	client := providerConf.DatadogClientV1
+	auth := providerConf.AuthV1
+	ddIndexList, _, err := client.LogsIndexesApi.GetLogsIndexOrder(auth).Execute()
```

Comment:
is this the same endpoint, just different named method? (Asking because I'm surprised cassettes didn't need re-recording)

Reply:
yeah it was using community client before, I just switched to ours
Also no cassettes cause no tests for this, I tested manually

Reply:
Ahh ok cool! 
Can you mention the client change in the Pr title so its called out in the changelog? Otherwise LGTM